sources/packages/web/src/components/generic/HeaderNavigator.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <slot name="title">
+    <div v-if="!routeName" class="form-header-title">{{ title }}</div>
+    <div v-if="routeName" class="form-header-title">
```

Comment:
The parameters are already in the props. No need to pass then as parameters.
the setup method can be declared as below
```javascript
setup(props:any){
  // Consume props directly, like props.routeName
}
```